lib/Parser/UrlCache.php

```diff
@@ -38,13 +38,13 @@ public function __construct() {
      * This file is encoded in JSON.
      */
     public function load() {
-        $privateDir = \Mesamatrix::$config->getValue("info", "private_dir", "private");
+        $privateDir = \Mesamatrix::$config->getValue("info", "private_dir");
```

Comment:
The way I see it, the private directory should always be set in the config, no exceptions. As such, there shouldn't be a default value.